### ISSUE TITLE
NOD | Downloadable copy of submission confirmation page

### ISSUE THREAD

**Issue by eileen-coforma** [department-of-veterans-affairs/va.gov-team#81507]
## Value Statement

**_As a_** Veteran that has just submitted an appeal on VA.gov
**_I want to_** download a copy of the information I submitted on the form
**_So that_** I can retain it for may records and reference it if I need to recall what I submitted for future filing

---

## Background Context

- MVF for downloadable copy of submission has been defined.

## Outcome, Success Measure, KPI(S), and Tracking Link

- CTA button click for NEW download event | Baseline: N/A | Target: TBD | Link to Data Source: TBD
- CTA button click for CURRENT print confirmation page event | Baseline: 89 clicks out of 564 submissions | Link to Data Source: TBD

## Acceptance Criteria
- [ ] User is advanced to a confirmation page where they can both see and save the information they submitted.
- [ ] User is aware they've successfully submitted their claim, when it was submitted, and what happens directly after submission.
- [ ] User can still access the confirmation page if there is a longer than acceptable delay in PDF generation.
- [ ] User can download a copy of the PDF via a link.
- [ ] User is notified if the PDF is not available for download.
- [ ] User can print the confirmation page by clicking a button and is made aware that printing will store all information on the page, but it won't be in the paper form format.
- [ ] User is made aware of the timeline for establishing their appeal and when/how to check the claim status tool. 
- [ ] Users are able to review all of their answers from the form, and get a list of evidence file names (not the actual files themselves) they've uploaded with the form.

## Designs and Build Notes

- [FIGMA](https://www.figma.com/file/BplQkEDZzD8NPPNmLrhvXv/Request-a-Board-Appeal-%2F-Notice-of-Disagreement-(VA-10182)?type=design&node-id=831-76228&mode=dev)

### Enablement team (if needed)
@ member of team

## Out of scope

- Confirmation email
- Supplemental claim and HLR confirmation pages

## Open questions

- Add here

## Tasks

- [ ] _Task_
- [ ] _Task_
- [ ] _Task_

## Definition of Ready

- [ ] Clear value description
- [ ] Testable acceptance criteria
- [ ] Accessibility added to acceptance criteria
- [ ] Approved designs attached
- [ ] Sample data provided where appropriate
- [ ] Estimated to fit within the sprint
- [ ] Dependencies and blockers linked

## Definition of Done

- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Reviewed and approved by product and/or design

**Comment by eileen-coforma** [comment]
@eileen-coforma confirm with Julie on timeout behavior.

**Comment by eileen-coforma** [comment]
Consider error state for PDF failure. Can it live outside of the blue box?

**Comment by eileen-coforma** [comment]
Blocked by LH endpoint availability

**Comment by dfong-adh** [comment]
No new fields, updates also on hold
